app/controllers/vm_common.rb

```diff
@@ -132,7 +132,7 @@ def launch_html5_console
     # override_content_security_policy_directives(:connect_src => ["'self'", "wss://#{request.env['SERVER_NAME']}"]
     override_content_security_policy_directives(:connect_src => ['*'], :img_src => %w(data: 'self'))
     password, host_address, host_port, _proxy_address, _proxy_port, protocol, ssl = @sb[:html5]
-
+    @vm = @record = identify_record(params[:id], VmOrTemplate)
```

Comment:
@arnikasky I think we only need @record to be present in vm_common/console_spice view, do we need to set @vm as well?